### PR TITLE
[flutter_conductor] Extend timeout for codesign integration test

### LINKED ISSUE #100937: [flutter_conductor] codesign_integration_test timing out after 5 minutes
I suspect this test was usually finishing in just under 5 minutes, as it needs to clone the framework and download cached binaries from scratch.

https://ci.chromium.org/ui/p/flutter/builders/prod/Mac%20framework_tests_misc/4046/overview

```
 05:03 +18 -1: test/codesign_integration_test.dart: validate the expected binaries from the conductor codesign command are present in the cache [E]                                                     
   TimeoutException after 0:05:00.000000: Test timed out after 5 minutes.
   dart:isolate  _RawReceivePortImpl._handleMessage
```

### PR DESCRIPTION
**Pull request by christopherfujino**
Fixes https://github.com/flutter/flutter/issues/100937

### CODE REVIEW

File: dev/conductor/core/test/codesign_integration_test.dart [hunk 1]
```diff
@@ -2,6 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// This test clones the framework and downloads pre-built binaries; it sometimes
+// times out with the default 5 minutes: https://github.com/flutter/flutter/issues/100937
+@Timeout(Duration(minutes: 10))
```
Comment (CaseyHillers) [inline dev/conductor/core/test/codesign_integration_test.dart:7]: It seems like the bot got throttled from GitHub and that's why this flakes.

It'd be nice if we could make conductor checkout against GoB, and then set the upstream back to GitHub.

**Review by CaseyHillers — APPROVED** [review]

**Review by goderbauer — APPROVED** [review]
LGTM

File: dev/conductor/core/test/codesign_integration_test.dart (hunk 1 above)
Comment (christopherfujino) [reply dev/conductor/core/test/codesign_integration_test.dart:7]: that wouldn't work pre-submit

Comment (CaseyHillers) [reply dev/conductor/core/test/codesign_integration_test.dart:7]: Wouldn't it work in this case? Or is the assumption this test might have a breaking change where you need the changes in presubmit? If so, we could copy the current framework checkout to a tmp folder.

**Pull request merged**